src/main/java/com/pikachu/purple/bootstrap/rating/api/RatingApi.java

```diff
@@ -0,0 +1,33 @@
+package com.pikachu.purple.bootstrap.rating.api;
+
+import com.pikachu.purple.bootstrap.common.security.Secured;
+import com.pikachu.purple.bootstrap.rating.dto.request.RatingCreateOnboardingRequest;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Tag(name = "rating", description = "perfume API")
```

Comment:
p2) description쪽에 수정사항 필요할 것 같아요~
```suggestion
@Tag(name = "Rating", description = "Rating API")
```